### PR TITLE
feat: curio docker devnet

### PR DESCRIPTION
**Pull request by LexLuthr**
Add instructions on how to create devnet images and spin of a docker devnet for Curio.

Merge after https://github.com/filecoin-project/lotus/pull/11954 is merged

### CODE REVIEW

**Review by Reiers — APPROVED** [review]
LGTM

File: content/en/storage-providers/curio/local-network.md [hunk 1]
```diff
@@ -12,6 +12,34 @@ weight: 140
...
+   ```
+
+2. Build the images
+   ```shell
+   make docker/curio-devnet
+   ```
+
+3. Start the docker devnet
+   ```shell
+   make curio-devnet/up
```
Comment (Reiers) [inline content/en/storage-providers/curio/local-network.md:32]: "sudo make curio-devnet/up" - it needs sudo to work.

File: content/en/storage-providers/curio/local-network.md [hunk 2]
```diff
@@ -12,6 +12,34 @@ weight: 140
...
+   ```
+
+4. Stop the docker devnet
+
+   ```shell
+   make curio-devnet/down
```
Comment (Reiers) [inline content/en/storage-providers/curio/local-network.md:38]: "sudo make curio-devnet/up" - it needs sudo to work.

**Review by Reiers — CHANGES_REQUESTED** [review]

**Comment by Reiers** [comment]
We don't need "sudo" - my local docker was the issue

**Review by snadrus — APPROVED** [review]

**Review by Reiers — APPROVED** [review]
Had my Infra test on their machines 👍 This guide is good 💯

**Pull request merged**